bin/si/src/args.rs

```diff
@@ -130,8 +129,7 @@ pub(crate) struct StatusArgs {
 }
 
 #[derive(Debug, clap::Args)]
-pub(crate) struct StartArgs {
-}
+pub(crate) struct StartArgs {}
```

Comment:
I noticed that the original change here didn't trigger a `check-format` for `bin/si`. I can look into this